plans/models.py

```diff
@@ -349,6 +349,21 @@ def create_for_users_without_plan(cls):
         return userplans
 
 
+@python_2_unicode_compatible
+class RecurringUserPlan(models.Model):
+    userplan = models.OneToOneField('UserPlan', on_delete=models.CASCADE, related_name='recurring')
+    token = models.CharField(_('Recurring token'), max_length=255, default=None, null=True, blank=True)
+    pricing = models.ForeignKey('Pricing', help_text=_('Recurring pricing'), default=None, null=True, blank=True, on_delete=models.CASCADE)
+    amount = models.DecimalField(
+        _('amount'), max_digits=7, decimal_places=2, db_index=True, null=True, blank=True)
+    tax = models.DecimalField(_('tax'), max_digits=4, decimal_places=2, db_index=True, null=True,
+                              blank=True)  # Tax=None is when tax is not applicable
+    currency = models.CharField(_('currency'), max_length=3, default='EUR')
```

Comment:
Maybe we can turn the default currency into a settings variable ?

Reply:
Anywhere else in `django-plans` is `EUR` as default.
Maybe we could make additional commit, that changes that behaviour in entire system, but I think, there might raise some problems with migrations - new migrations would be generated on settings change.

Anyway, I don't think, that it is necessary to have any default for `RecurringUserPlan` at all. That object is created only with new order, so the currency should be always known. I will leave it with no default and test if it does not create any problems.